src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -55,21 +54,28 @@ def __init__(self, provider_configurations, app=None):
         self.clients = None
         self._logout_view = None
         self._error_view = None
+        self._redirect_uri_endpoint = 'redirect_uri'
 
         if app:
             self.init_app(app)
 
     def init_app(self, app):
+        if 'OIDC_REDIRECT_ENDPOINT' in app.config:
+            redirect_endpoint = app.config.get('OIDC_REDIRECT_ENDPOINT')
+            if redirect_endpoint[0] == '/':
```

Comment:
I think this can be more permissive and allow the leading `/`, simply removing it if it's there. Then reading the value from config can also be simplified as:
```python
self._redirect_uri_endpoint = app.config.get('OIDC_REDIRECT_ENDPOINT', 'redirect_uri').lstrip('/')
```

With this the initialisation in the constructor should be changed to `self._redirect_uri_endpoint = None` to avoid duplicating the default value.